modules/table-bucket/variables.tf

```diff
@@ -10,10 +10,16 @@ variable "table_bucket_name" {
   default     = null
 }
 
+variable "encryption_configuration" {
+  description = "Map of encryption configurations"
+  type        = any
+  default     = null
+}
+
 variable "maintenance_configuration" {
   description = "Map of table bucket maintenance configurations"
   type        = any
-  default     = {}
+  default     = null
```

Comment:
Turns out if you don't specify `maintenance_configuration` and rely on this default value, an error occurs unless `null`. 
```
╷
│ Error: Incorrect attribute value type
│ 
│   on ../../modules/table-bucket/main.tf line 6, in resource "aws_s3tables_table_bucket" "this":
│    6:   maintenance_configuration = var.maintenance_configuration
│     ├────────────────
│     │ var.maintenance_configuration is object with no attributes
│ 
│ Inappropriate value for attribute "maintenance_configuration": attribute "iceberg_unreferenced_file_removal" is required.
╵
```